.github/workflows/ci.yml

```diff
@@ -7,14 +7,17 @@ on:
     branches: [main]
   pull_request:
     branches: [main]
+    types: [opened, synchronize, reopened, ready_for_review]
```

Comment:
I'm guessing this was to try and prevent pipeline from running on draft PRs? However, @rajatagarwal-ibm made the same change in mongoDB module, and it doesn't work. I created a draft PR and it ran the pipeline. See https://github.com/terraform-ibm-modules/terraform-ibm-icd-mongodb/pull/153

Reply:
As per scrum discussions, further pipeline changes required for this to work